### PR TITLE
Use priority 200 for uninstalling the module BZ(1777740)

### PR DESCRIPTION
**Pull request by zpytela**
Change the uninstall command in the %postun section of the specfile
to use the %selinux_modules_uninstall macro which uses priority 200.
Change formatting so that the lines are vertically aligned
in the %postun section.

### CODE REVIEW

File: contrib/container-selinux.spec [hunk 1]
```diff
@@ -107,11 +107,11 @@ fi
 
 %postun
 if [ $1 -eq 0 ]; then
-%{_sbindir}/semodule -n -r %{modulenames} docker &> /dev/null || :
-if %{_sbindir}/selinuxenabled ; then
-%{_sbindir}/load_policy
-%relabel_files
-fi
+   %selinux_modules_uninstall -s %{selinuxtype} %{modulename} docker
+   if %{_sbindir}/selinuxenabled ; then
```
Comment (wrabcak) [inline contrib/container-selinux.spec]: Lines from 111 to 114 are already in the rpm macro.

File: contrib/container-selinux.spec [hunk 2]
```diff
@@ -96,7 +96,7 @@ fi
 %{_sbindir}/semodule -n -s %{selinuxtype} -r container 2> /dev/null
 %{_sbindir}/semodule -n -s %{selinuxtype} -d docker 2> /dev/null
 %{_sbindir}/semodule -n -s %{selinuxtype} -d gear 2> /dev/null
-%{_sbindir}/semodule -n -X 200 -s %{selinuxtype} -i $MODULES > /dev/null
+%selinux_modules_install -s %{selinuxtype} $MODULES
 if %{_sbindir}/selinuxenabled ; then
```
Comment (wrabcak) [inline contrib/container-selinux.spec]: Load policy process is in macro.

File: contrib/container-selinux.spec (hunk 1 above)
Comment (zpytela) [reply contrib/container-selinux.spec]: I can't see %relabel_files equivalent in the macro.

Comment (wrabcak) [reply contrib/container-selinux.spec]: there is %selinux_relabel_pre and %selinux_relabel_post which has improved functionality and different design

**Comment by wrabcak** [comment]
@zpytela , Do you have also working scratch build? 

Thanks,
Lukas.

**Comment by rhatdan** [comment]
LGTM
This internel spec file is not the one being used in Fedora or RHEL, We need to change these in Kohi and Bodhi.

**Comment by rhatdan** [comment]
I have updated container-selinux package in Rawhide to use the selinux macros.

**Pull request merged**